### DISCUSSION TITLE
Portal content backups with the built-in Git repository

### DISCUSSION THREAD

**Discussion by santo2** [Azure/api-management-developer-portal#2590] [Feature request]
Hi,

APIM allows us to copy paste the full APIM data to another instance, the old portal is included here, but the new one doesn't seem to be in this repository anymore. How do you suggest doing staged deployments? 
We copied the resources from our TEST to our ACC and then PROD environment for api management deployments.

**Comment by ghost** [comment]
@santo2, thank you for opening this issue. We will triage it within the next few business days.

**Comment by santo2** [comment]
I found this question from more than a year ago and it wasn't answerd either.
https://feedback.azure.com/d365community/idea/5a5395f6-0025-ec11-b6e6-000d3a4f0858

**Comment by ghost** [comment]
@santo2, thank you for requesting this feature.

**Comment by azaslonov** [comment]
Hello @santo2, you're right, this is a known gap that we need to fill. For now, the work around is to move the dev portal content using [this script](https://github.com/Azure/api-management-developer-portal/blob/master/scripts.v3/migrate.bat).

**Comment by mikebudzynski** [comment]
Thorough documentation is here: https://docs.microsoft.com/en-us/azure/api-management/automate-portal-deployments

**Comment by ghost** [comment]
@santo2, thank you for requesting this feature.
